distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -980,6 +980,25 @@ type Vector
         json = this.take_start 100 . to_json
         json.to_text
 
+    ## PRIVATE
+       This is a workaround for a bug, should be removed once the bug is fixed.
+       https://www.pivotaltracker.com/story/show/181057718
+
+       It also ensures correct warning propagation which is another bug.
```

Comment:
ticket?

Reply:
I think I will just remove this, I currently do not use that and it is a band-aid fix for a set of deeper errors.